ibis/pandas/api.py

```diff
@@ -7,3 +7,19 @@
 
 def connect(dictionary):
     return PandasClient(dictionary)
+
+
+def from_dataframe(df):
```

Comment:
Let's also add a `client=None` parameter and a `name='df'` parameter.

Reply:
done